buildSrc/src/main/kotlin/pg-index-health.java-conventions.gradle.kts

```diff
@@ -95,3 +96,9 @@ spotbugs {
     reportLevel.set(Confidence.LOW)
     excludeFilter.set(file("../config/spotbugs/exclude.xml"))
 }
+
+forbiddenApis {
```

Comment:
It should be only in one place (in buildSrc/src/main/kotlin/pg-index-health.forbidden-apis.gradle.kts)